### PR TITLE
Add minutes from last meeting

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: src/orga/minutes/2020-10-14.md [hunk 1]
```diff
@@ -0,0 +1,896 @@
...
+    * Public or internal? I don't really care either way. Anyone feel strongly one way or the other?
+    * opt-in to public bios
+    * software carpentry has a large number of instructors and has https://carpentries.org/instructors
+    * some concern about "yet another place to keep stuff up to date"
+* [ ] (ED) document strategies for reproducible environments using conda-forge
+* [ ] (UK) Static libraries stuff
+    * [ ] Add linting hints to builds to find them
+    * [x] Recommend how to package them -> CFEP-18
+    * [x] We should write docs saying we don't provide support and this is a bad idea. -> CFEP-18
+# 2020-10-14 conda-forge core meeting
```
Comment (beckermr) [inline src/orga/minutes/2020-10-14.md:225]: Why are there two copies?

**Review by beckermr — CHANGES_REQUESTED** [review]

File: src/orga/minutes/2020-10-14.md (hunk 1 above)
Comment (jakirkham) [reply src/orga/minutes/2020-10-14.md]: I pushed a commit to drop this

**Comment by jakirkham** [comment]
Do we also have the minutes from the last meeting somewhere?

**Review by beckermr — APPROVED** [review]

**Comment by jakirkham** [comment]
Added notes from the last two meetings here: https://github.com/conda-forge/conda-forge.github.io/pull/1177

**Pull request merged**